app/assets/javascripts/submissions.coffee

```diff
@@ -270,7 +270,7 @@ class TrialSubmission extends Submission
         @$('.uploaded-layout .delete-layout-upload').attr(href: data.result.delete_url)
 
         @$('.uploaded-layout .layout-image').prop(src: data.result.small_file_url)
-        @$('.uploaded-layout .layout-image').parent().prop(href: data.result.original_file_url)
+        @$('.uploaded-layout .layout-image').parent('a').prop(href: data.result.original_file_url)
```

Comment:
This change looks suspicious - was it needed? AFAIR `parent()` traverses one level up, if you pass selector as an argument it returns empty set unless parent matches that selector.